main.go

```diff
@@ -153,6 +153,13 @@ func main() {
 			Usage:       "Declare specific feature values on start up. Example: \"kontainer-driver=true\" - kontainer driver feature will be enabled despite false default value",
 			Destination: &config.Features,
 		},
+		cli.StringFlag{
+			Name:        "extension-api-app-selector",
+			EnvVar:      "EXTENSION_API_APP_SELECTOR",
+			Value:       "rancher",
+			Usage:       "Defines the value to use the constructing the label selector for the imperative api APIService",
+			Destination: &config.ExtensionOpts.AppSelector,
+		},
```

Comment:
We should be fine without a CLI argument for this. This will only be needed for the Helm case really, as docker / binary use a selector pointing to RDP.

Reply:
I agree we don't *need* a cli here, but I like the clarity of having a clear path of a value from cli to its usage. I'll update to using an envar here since rancher uses envvars a lot. I'll keep the  `ext.Options` since it'll make it easier to test the ext packages if/when we start adding more tests.